common/utils/analytics.js

```diff
@@ -120,7 +128,7 @@ export default ({ category, contentType, pageState, featuresCohort }: Props) =>
             action: 'Percentage',
             label: mark,
             value: 1,
-            nonInteraction: true
+            nonInteraction: Boolean(timing < 10000)
```

Comment:
Could we not set a state on the `Time on page >=` that changes as these are intrinsically coupled?